### ISSUE TITLE
deploy error with conjure-up kubernetes

### ISSUE THREAD

**Issue by evilnick** [conjure-up/conjure-up#1537]
## Report

Error when doing a plain conjure-up kubernetes to AWS

Using juju 2.4.5-xenial-amd64 from snap/stable
conjure-up stable and edge versions report the same error

the install log is [here](https://paste.ubuntu.com/p/dSZyXP3mbs/) but the relevant part seems to be:

```
2018-11-07 15:42:41,543 [DEBUG] conjure-up/canonical-kubernetes - events.py:52 - Setting Error at conjureup/events.py:149
2018-11-07 15:42:41,543 [ERROR] conjure-up/canonical-kubernetes - events.py:161 - Unhandled exception in <Task finished coro=<do_deploy() done, defined at /snap/conjure-up/1031/lib/python3.6/site-packages/conjureup/controllers/juju/deploy/common.py:11> exception=TypeError('deploy() takes 9 positional arguments but 11 were given',)>
Traceback (most recent call last):
File "/snap/conjure-up/1031/lib/python3.6/site-packages/conjureup/controllers/juju/deploy/common.py", line 30, in do_deploy
await app.juju.client.deploy(fn)
File "/snap/conjure-up/1031/lib/python3.6/site-packages/juju/model.py", line 1283, in deploy
await handler.execute_plan()
File "/snap/conjure-up/1031/lib/python3.6/site-packages/juju/model.py", line 1947, in execute_plan
result = await method(*step.args)
TypeError: deploy() takes 9 positional arguments but 11 were given
```

**Comment by seffyroff** [comment]
I'm seeing the same thing here.  Full log is [here](https://pastebin.ubuntu.com/p/PMqHPpgZNd/).

**Comment by battlemidget** [comment]
This is an issue with latest beta/edge juju as it has some modifications for k8s deployments. @johnsca Did you ever get information from juju team about this?

**Comment by kwmonroe** [comment]
Fixed in conjure-up 2.6.5
